Tribler/Test/Community/Multichain/StatisticsDispaly/test_statistics_database.py

```diff
@@ -0,0 +1,73 @@
+from Tribler.Test.test_as_server import BaseTestCase
+from Tribler.community.multichain.StatisticsDisplay.networkNode import NetworkNode
+from Tribler.community.multichain.StatisticsDisplay.dbDriver import DbDriver
+
+from twisted.internet.defer import inlineCallbacks
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+
+class TestStatisticsDatabase(BaseTestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestStatisticsDatabase, self).__init__(*args, **kwargs)
+
+    @blocking_call_on_reactor_thread
+    @inlineCallbacks
+    def setUp(self, **kwargs):
+        yield super(TestStatisticsDatabase, self).setUp()
+        self.driver = DbDriver()
+        self.focus_node = NetworkNode("30", self.driver)  # hardcoded hex of focus node
+
+
+    @blocking_call_on_reactor_thread
+    def test_get_neighbors(self):
+        # Arrange
+        expected_result = ["31", "32", "33", "34"]
+        # Act
+        result_list = self.focus_node.neighbor_keys
+        # Assert
+        assert len(result_list) == len(expected_result)
+        for key_hash in expected_result:
+            if key_hash not in result_list:
+                assert False, "Neighbor was not in list: %s" % key_hash
+
+    @blocking_call_on_reactor_thread
+    def test_total_up(self):
+        # Act
```

Comment:
Act marks the part of the test where you execute an action. For explanation, see my previous comment.